DirectProgramming/C++SYCL_FPGA/Tutorials/Features/kernel_args_restrict/README.md

```diff
@@ -1,14 +1,24 @@
-# Avoiding Aliasing of Kernel Arguments
+# `Kernel Args Restrict` Sample
```

Comment:
Should we follow the scheme of have this line being: 
`name of the feature` Sample?

In this case that would be:
```suggestion
# `kernel_args_restrict` Sample
```

Let me know what you think about it